### PR TITLE
Update README.md for Spring 2022 to include Jessica Wang

### PR DESCRIPTION
**Pull request by jewang25**
**Summary**
- This pull request added Jessica Wang to README for Spring 2022

### CODE REVIEW

**Review by CollinWoo — COMMENTED** [review]
This mostly looks good! But I think the name order is slightly off...  https://github.com/cornell-dti/zing-lsc/pull/18 states that names should be ordered by role, then alphabetically by first name and then last name, so your name should go right after Jessica Chai's. Otherwise it looks perfect to me!

**Comment by jewang25** [comment]
> This mostly looks good! But I think the name order is slightly off... #18 states that names should be ordered by role, then alphabetically by first name and then last name, so your name should go right after Jessica Chai's. Otherwise it looks perfect to me!

Ok thanks! I just changed it.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,10 @@
 - **Alyssa Zhang** - Designer
 - **Grace Han** - Developer
 - **Jessica Chai** - Developer
+- **Jessica Wang** - Developer
 - **Miranda Luo** - Developer
 
+
```
Comment (Coreball) [inline README.md]: Extra newline?

**Review by Coreball — APPROVED** [review]
Welcome to Zing!

File: README.md (hunk 1 above)
Comment (jewang25) [reply README.md]: Sorry, I just removed it.

**Pull request merged**